.github/pull_request_template.md

```diff
@@ -1,49 +1,53 @@
-<!--
-  Your description goes here: A summary of the change, including any relevant motivation and context.
+## What it does
 
-  If relevant, include a description both of the existing behavior and of the new behavior.
+<!--
+  A summary of the change, including any relevant background, motivation, and context.
+  If relevant, include a description, screenshots, and/or video of the existing and new behavior.
 -->
 
+## Links
+
 <!--
-  Other aspects to consider. uncomment and add detail for any that seem necessary:
+  Links to relevant external resources; ie, specification documents, Jira tickets, related PRs, Honeybadger errors, etc.
 -->
 
-<!-- ### Background -->
-<!-- ### Privacy -->
-<!--
-1.	Does the Project involve the collection, use or sharing of new Personal Data?
+## Testing story
 
-2.	Does the Project involve a new or changed use or sharing of existing Personal Data?
--->
-<!-- ### Security -->
 <!--
-Link to Jira Task where sensitive security issues are discussed privately.
+  Does your change include appropriate tests?
+  If so, please describe how the tests included in this PR are sufficient.
+  If not, please explain why this change does not need to be tested.
 -->
-<!-- ### Caching -->
-<!-- ### Deployment strategy -->
-<!-- ### Future work -->
 
-## Links
+<!-- Other aspects to consider. Delete any sections that are not relevant to your change. -->
+
+## Privacy
 
 <!--
-  Any relevant links to external resources; ie, specification documents, jira
-  items, related PRs, honeybadger errors, etc
+  1.	Does this change involve the collection, use, or sharing of new Personal Data?
+  2.	Does this change involve a new or changed use or sharing of existing Personal Data?
 -->
 
-- [spec]()
-- [jira]()
+## Security
 
-## Testing story
+<!-- Link to Jira task(s) where sensitive security issues are discussed privately. -->
 
-<!--
-  Does your change include appropriate tests?
+## Caching
 
-  If so, please describe how the tests included in this PR are sufficient
+## Deployment strategy
 
-  If not, please explain why this change does not need to be tested.
+## Follow-up work
```

Comment:
"Deployment strategy" and "follow-up work" could be worth moving up to just after "Testing". I've definitely used these sections, but I still don't think I've used Privacy, Security or Caching. (it's possible I've forgotten about these sections and just talked about them in the background/description)

Reply:
yeah, i always end up putting that info in the background as well. these headers won't be commented out by default anymore, so i want to see if people are using them for a bit, but we might consider removing those headers in favor of comments within "what it does" at some point

Reply:
oh, and agreed that "deployment strategy" and "follow-up work" should move up -- i'm moving those now